content/docs/get-started-with-neon/using-api-keys.md

```diff
@@ -12,8 +12,94 @@ Once authenticated, a user can create and access Projects and [query Project dat
 
 ## Using API keys as a Neon User
 
-API keys allow users to access Neon application programming interface.
+API keys allow users to access the Neon application programming interface.
 
-An API key provides access to any action available to the user. Currently, API keys cannot be scoped to the specific Projects. Neon users can provision multiple API keys. An API key that is no longer needed can be revoked; this action cannot be reverted. Any issued API key is valid forever until it is revoked.
+An API key provides access to any action available to the user.An API key that is no longer needed can be revoked; this action cannot be reverted. Any issued API key is valid forever until it is revoked. Neon users can generate multiple API keys.
+
+### Issue a New API Key
+
+Here's how to issue a new API key:
+
+1. Start by logging in to [Neon Console](https://console.neon.tech).
+2. Click on your username in the upper right corner, then click the `Settings` button.
+3. Under `Developer Settings` click on `Generate new API Key`.
+4. Choose a unique name that will help you remember what this key is for. For example, if you want a key for your testing environment, name the key `Test Key`.
+5. Click the `Create` button and copy the generated key.
+
+You need to store your key in a safe location, Neon does not store this key for you. If you lose your key, revoke the lost key and create a new key to access the Neon API. You can safely store your API key in a credential manager on your local machine, or using a credential management service like [AWS Key Management Service](https://aws.amazon.com/kms/).
+
+Remember, all API keys remain valid until you revoke them.
+
+### Revoke an API Key
+
+1. In your Neon Console, click on your username in the upper right corner and click the `Settings` button.
+2. Under `Developer Settings` you will see the list of issued and active API keys.
+3. To revoke the key forever, click the `Revoke` button. This will immediately revoke the key, all the requests using this key will now fail.
+
+### Making API Calls
+
+Every request to the Neon API endpoints should pass an API key in the `Authorization` HTTP header. You can see the available endpoints in our [API Reference](https://console.neon.tech/api-docs/v1).
+
+Let’s look at how to make a `curl` request using your Neon API key. We're going to use the `projects` endpoint to get a list of the project IDs in an account.
```

Comment:
Actually, we're getting more info, not just the IDs. 
```suggestion
Let’s look at how to make a `curl` request using your Neon API key. We're going to use the `projects` endpoint to get a list of the projects in an account.
```
WDYT about this suggestion?

Reply:
This makes the sentence a lot clearer